tests/test_layout.py

```diff
@@ -39,3 +39,72 @@ def test_that_product_older_versions_are_ordered_correctly(self, mozwebqa):
         csp = CrashStatsHomePage(mozwebqa)
 
         Assert.is_sorted_descending(csp.header.other_versions)
+
+
+class TestSuperSearchLayout:
```

Comment:
Just wanted to experiment with the idea. I think it has the opportunity to make the tests more readable.